### ISSUE TITLE
Video in OTR Freezes when switching away from the source window of the playing video.

### ISSUE THREAD

**Issue by mcolinp** [LorenzCK/OnTopReplica#154]
I vaguely remember dealing with this once before, but I don't remember the fix. I primarily watch Youtube videos using Firefox. When using OTR, I typically use the "Region" option to remove unwanted ui elements around the video. Not sure if this contributes to the issue or not. I vaguely remember there  potentially being something I needed to change with Firefox. I saw another issue with Chrome mentioning flags, but I'm not sure if that is the same with Firefox, and how to make and save the needed changes to get video working.

**Comment by epiloso** [comment]
This got me searching the same issue w/ Firefox... found this thread on [Reddit](https://www.reddit.com/r/incremental_games/comments/re9vic/psa_the_next_version_of_firefox_96_will_disable/) about the "occlusion" setting but here is the TLDR... the tab still can't be minimized but the video won't freeze when the tab is behind another window.

Go to about:config in the browser address bar
Search for dom.timeout.enable_budget_timer_throttling and set value to false
Search for widget.windows.window_occlusion_tracking.enabled and set value to false
Search for dom.min_background_timeout_value_without_budget_throttling and set value to 10

**Comment by peterader** [comment]
Just came across this having a similar issue in Chrome. When the replicated window went to background the video stopped playing and OTR went white. To fix this in Chrome: Enter this in the address bar: chrome://flags hit enter, then search calculate-native-win-occlusion. Change it to disabled and restart chrome.

**Comment by paulbenson88** [comment]
> Just came across this having a similar issue in Chrome. When the replicated window went to background the video stopped playing and OTR went white. To fix this in Chrome: Enter this in the address bar: chrome://flags hit enter, then search calculate-native-win-occlusion. Change it to disabled and restart chrome.

Is there any reason why I can't seem to find that specific setting within chrome://flags?

**Comment by peterader** [comment]
> Is there any reason why I can't seem to find that specific setting within chrome://flags?

You are right, looks like the setting has been removed. I have chrome 131.0.6778.109 and can't see it either. However, the issue is not present any more either. Maybe it was fixed.

**Comment by aacid** [comment]
> This got me searching the same issue w/ Firefox... found this thread on [Reddit](https://www.reddit.com/r/incremental_games/comments/re9vic/psa_the_next_version_of_firefox_96_will_disable/) about the "occlusion" setting but here is the TLDR... the tab still can't be minimized but the video won't freeze when the tab is behind another window.
> 
> Go to about:config in the browser address bar Search for dom.timeout.enable_budget_timer_throttling and set value to false Search for widget.windows.window_occlusion_tracking.enabled and set value to false Search for dom.min_background_timeout_value_without_budget_throttling and set value to 10

this worked perfectly on **firefox**! didn't even needed restart, neither firefox or OTR, worked right away.
Thanks
